Только в livecd-tools-031.orig: debugfiles.list
Только в livecd-tools-031.orig: debuglinks.list
Только в livecd-tools-031.orig: debugsources.list
Только в livecd-tools-031.orig/docs: livecd-creator.8
Только в livecd-tools-031.orig/docs: livecd-iso-to-disk.8
Только в livecd-tools-031/imgcreate: creator.py~
diff -ur livecd-tools-031.orig/imgcreate/kickstart.py livecd-tools-031/imgcreate/kickstart.py
--- livecd-tools-031.orig/imgcreate/kickstart.py	2010-09-20 15:11:26.000000000 +0400
+++ livecd-tools-031/imgcreate/kickstart.py	2010-09-27 17:01:35.330554335 +0400
@@ -24,10 +24,10 @@
 import logging
 import urlgrabber
 
-try:
-    import system_config_keyboard.keyboard as keyboard
-except ImportError:
-    import rhpl.keyboard as keyboard
+#try:
+    #import system_config_keyboard.keyboard as keyboard
+#except ImportError:
+    #import rhpl.keyboard as keyboard
 
 import pykickstart.commands as kscommands
 import pykickstart.constants as ksconstants
@@ -135,10 +135,16 @@
 class KeyboardConfig(KickstartConfig):
     """A class to apply a kickstart keyboard configuration to a system."""
     def apply(self, kskeyboard):
-        k = keyboard.Keyboard()
+        #k = keyboard.Keyboard()
+
         if kskeyboard.keyboard:
-            k.set(kskeyboard.keyboard)
-        k.write(self.instroot)
+            f = open(self.path("/etc/sysconfig/keyboard"), "w+")
+            f.write("KEYBOARD=\"" + kskeyboard.keyboard + "\"\n")
+            f.close()
+
+        #if kskeyboard.keyboard:
+            #k.set(kskeyboard.keyboard)
+        #k.write(self.instroot)
 
 class TimezoneConfig(KickstartConfig):
     """A class to apply a kickstart timezone configuration to a system."""
Только в livecd-tools-031/imgcreate: kickstart.py~
